app/mutations/FormChange/discardFormChange.ts

```diff
@@ -0,0 +1,9 @@
+import { graphql } from "react-relay";
+
+export const mutation = graphql`
+  mutation discardFormChangeMutation($input: UpdateFormChangeInput!) {
```

Comment:
Should this be removed for now? We can leave it if we're sure it will be used, but I don't want to introduce dead code.

Reply:
It's part of the AC of [the card](https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/96) so I left it there. It's not needed right now, but we'll most likely need it after today's discussion